### DISCUSSION TITLE
[Next 13 Beta (App)] Vercel favicon keeps loading after custom favicon

### DISCUSSION THREAD

**Discussion by dimib-tech** [vercel/next.js#46548] [Help] [answered]
# Has Beta App Directory Enabled

## Problem:
I've got a website hosted on Vercel with Next 13 ([here](https://dimib.tech)) and it keeps loading vercels favicon right after my favicon, overriding it as a result, and only showing vercels default favicon in the tab. 

## Steps to reproduce
1) init a fresh next 13 app with typescript,eslint and app dir enabled.
2) Set up favicon via the manifest api (more info [here](https://beta.nextjs.org/docs/api-reference/metadata#icons))

Here is my code:
```
icons: {
		icon: { url: "/icon.png", type: "image/x-icon", sizes: "any" },
		others: [
			{ url: "/favicon-32x32.png", type: "image/png", sizes: "32x32" },
			{ url: "/favicon-16x16.png", type: "image/png", sizes: "16x16" },
		],
	}
```

I've tried multiple different combinations including:
```
icons:{
icon:'favicon.png/.ico'
}
```
And I'm all out of ideas on how to fix this.
![image](https://user-images.githubusercontent.com/59563446/221850735-56967583-d98a-46fc-a063-0df5bed478ba.png)

I'll  be glad to post additional info on this topic, so ask me anything. And if we conclude it's a bug, submit a bug report.

**Comment by ciruz** [comment, marked as answer]
When I try to open your favicon, the `icon.png` does not exist.

Take a look => https://www.dimib.tech/icon.png gives a 404.

It's also funny, that the 404 site, has the correct favicon.

![image](https://user-images.githubusercontent.com/987947/221867547-f36b5c07-4409-4558-a028-3d4c27189a9d.png)

My thoughts:
1. the favicon on the 404 works correct, because a true default favicon exists: https://www.dimib.tech/favicon.ico , so the browser uses this on the 404 page, since there is no favicon defined.
2. the favicon you have defined in your app, does not work since its invalid / does not exist, therefore the browser just loads another favicon
3. you have multiple favicons defined, so the browser uses just the next (better) one, keep in mind that the classic `favicon.ico` will always be the last choice of the browser, because of the poor image quality:
 ![image](https://user-images.githubusercontent.com/987947/221868622-eb4eea77-958c-4035-a76f-7559ae726436.png)

In your case the next better favicon is the third one, from `/_next/static/` which is the favicon you see on your site:
![image](https://user-images.githubusercontent.com/987947/221868877-5df14702-f52d-439e-b9cd-4bcf4f025562.png)

**Reply by dimib-tech** [reply]
Welp, my dumba$$ misplaced the favicon into the '/' instead of deleting it. Joys of using a touchpad to code i guess... Thank you kind stranger!

Also, that 404 behaviour really threw me off.
